proposals/windows-mdm-commands-tables.md

```diff
@@ -0,0 +1,118 @@
+# Changes to Windows MDM command tables
+
+**Context** Commands are the mechanism we'll use to build features like
+
+- Deliver OS settings
+- Filter hosts by their "OS settings" status (pending, verified, verifying, failed)
+
+IF possible I would like us to have the same mental model for both platforms,
+where we have three tables with a clear separation of concerns:
+
+A. List of commands
+B. Queue
+C. List of command results
+
+Any command, doesn't matter how it was enqueued (via the UI/CLI or in response
+to other commands) should have an entry in `A`, plus entries with FK references
+in `B` when it's enqueued, and `C` when we get results.
```

Comment:
Yeah I agree with that, it echoes the concern I had in the original PR: https://github.com/fleetdm/fleet/pull/14134#discussion_r1337194086